commands/sketch/new.go

```diff
@@ -59,3 +69,20 @@ func NewSketch(ctx context.Context, req *rpc.NewSketchRequest) (*rpc.NewSketchRe
 
 	return &rpc.NewSketchResponse{MainFile: sketchMainFilePath.String()}, nil
 }
+
+func validateSketchName(name string) error {
+	if name == "" {
+		return &arduino.CantCreateSketchError{Cause: errors.New(tr("sketch name cannot be empty"))}
+	}
+	if len(name) > sketchNameMaxLength {
+		return &arduino.CantCreateSketchError{Cause: errors.New(tr("sketch name too long (%d characters). Maximum allowed length is %d",
```

Comment:
`"sketch name too long (%d characters). Maximum allowed length is %d"`
This string has 2 positional parameters, you must replace `%d` with `%[1]d` and `%[2]d`.
This would normally not be a problem, but since it's translated into another language with `tr(...)`, the ordering of the parameters is not guaranteed to stay the same.